package-lock.json

```diff
@@ -0,0 +1,6162 @@
+{
```

Comment:
Any reason why you decided to switch from yarn to npm in this project?

Reply:
Nope, accidental. I had a plugin installed on vscode that used npm by default. I didn't notice I committed it. It is removed